internal/controller/active_connection_registrar.go

```diff
@@ -32,6 +33,14 @@ type cancellationMap struct {
 	sync.RWMutex
 }
 
+func GetTickerInterval(cfg *config.Config) (duration time.Duration) {
```

Comment:
Let's make this a private function for now.

I think this will give us a more random interval:

```
	min := cfg.GatewayActiveConnectionRegistrarPollMinDelay
	max := cfg.GatewayActiveConnectionRegistrarPollMaxDelay
	value := rand.Intn(max-min+1) + min

```

Here's a playground to try it out:
https://play.golang.org/p/GUS33v2nMd_9

Reply:
I made it public so the the tests could call it. Is there a better way to do that?

Reply:
Currently, the tests are in the same package (the controller package), so this method doesn't have to be public.